Description: Avoid building libXcomp.so.1 (which is identical to libXcomp.so.3)
Author: Orion Poplawski <orion@cora.nwra.com>

--- a/nx-X11/lib/X11/Imakefile	2013-01-04 05:25:49.000000000 -0700
+++ b/nx-X11/lib/X11/Imakefile	2013-01-15 15:26:07.969167259 -0700
@@ -63,7 +63,6 @@ NX_XCOMPLIBDIR    = $(XTOP)/../nxcomp
 NX_XCOMPLIBLINK   = Xcomp
 NX_XCOMPLIBTARGET = $(NX_XCOMPLIBDIR)/$(NX_XCOMPLIBNAME)
 NX_REQUIREDLIBS   = -L$(NX_XCOMPLIBDIR) -l$(NX_XCOMPLIBLINK)
-NX_XCOMPDEPTARGET = $(BUILDLIBDIR)/$(NX_XCOMPLIBNAME)
 NX_XCOMPCONFIGTARGET = $(NX_XCOMPLIBDIR)/config.status
 
 NX_XCOMPEXTLIBDIR    = $(XTOP)/../nxcompext
@@ -1181,17 +1180,9 @@ lib$(LIBNAME).so.$(SOXLIBREV): $(NX_XCOM
 
 all:: $(NX_XCOMPLIBTARGET)
 
-$(NX_XCOMPDEPTARGET):
-	ln -s $(NX_XCOMPLIBDIR)/$(NX_XCOMPLIBNAME) $(BUILDLIBDIR)/$(NX_XCOMPLIBNAME)
-	ln -s $(NX_XCOMPLIBDIR)/$(NX_XCOMPLIBNAME).1 $(BUILDLIBDIR)/$(NX_XCOMPLIBNAME).1
-
 depend:: $(NX_XCOMPLIBTARGET)
 
-all:: $(NX_XCOMPDEPTARGET) $(NX_XCOMPEXTLIBTARGET) 
-
-clean::
-	rm -f $(BUILDLIBDIR)/$(NX_XCOMPLIBNAME)
-	rm -f $(BUILDLIBDIR)/$(NX_XCOMPLIBNAME).1
+all:: $(NX_XCOMPEXTLIBTARGET) 
 
 #endif
 
